Sources/SectionScrubber.swift

```diff
@@ -121,13 +121,13 @@ public class SectionScrubber: UIView {
 
         self.dragGestureRecognizer.addTarget(self, action: #selector(self.handleScrub))
         self.dragGestureRecognizer.delegate = self
-        self.addGestureRecognizer(self.dragGestureRecognizer)
+        self.scrubberImageView.addGestureRecognizer(self.dragGestureRecognizer)
 
         self.longPressGestureRecognizer.addTarget(self, action: #selector(self.handleScrub))
         self.longPressGestureRecognizer.minimumPressDuration = 0.2
         self.longPressGestureRecognizer.cancelsTouchesInView = false
         self.longPressGestureRecognizer.delegate = self
-        self.addGestureRecognizer(self.longPressGestureRecognizer)
+        self.scrubberImageView.addGestureRecognizer(self.longPressGestureRecognizer)
```

Comment:
I see you've added the gesture recogniser to the scrubberImageView, but this imageView is only 26*56 and because of the small width still hard to touch sometimes. I noticed this behaviour when I was working on it earlier and thats why I added an *extra* special gesture-view of 56x44. I worked really well!